static/src/javascripts/projects/common/modules/experiments/tests/february-moment-banner-non-us.js

```diff
@@ -38,6 +39,7 @@ export const februaryMomentBannerNonUS: AcquisitionsABTest = {
                 bannerModifierClass: 'fiv-banner',
                 // minArticlesBeforeShowingBanner?: number,
             },
+            canRun: () => canShowBannerSync(),
```

Comment:
let's just leave BIG comments saying how you must pass in `minArticlesBeforeShowingBanner` and `userCohort` if you've got these set in the `engagementBannerParams`

since this will probably be copy-pasted for other tests

Reply:
yep good point. This is going to get really nasty